test/javascripts/acceptance/search-full-test.js.es6

```diff
@@ -190,19 +193,55 @@ test("update category through advanced search ui", assert => {
 //   });
 // });
 
-test("update in filter through advanced search ui", assert => {
+test("update in:likes filter through advanced search ui", assert => {
   visit("/search");
   fillIn('.search input.full-page-search', 'none');
   click('.search-advanced-btn');
-  selectDropdown('.search-advanced-options #s2id_in', 'likes');
-  fillIn('.search-advanced-options #in', 'likes');
+  click('.search-advanced-options .in-likes');
 
   andThen(() => {
-    assert.ok(exists('.search-advanced-options #s2id_in .select2-choice .select2-chosen:contains("I liked")'), 'has "I liked" populated');
+    assert.ok(exists('.search-advanced-options .in-likes:checked'), 'has "I liked" populated');
     assert.equal(find('.search input.full-page-search').val(), "none in:likes", 'has updated search term to "none in:likes"');
   });
 });
 
+test("update in:private filter through advanced search ui", assert => {
```

Comment:
Great job on the acceptance tests :+1: It makes refactoring so much easier :smile: